task_sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -173,9 +177,8 @@ def run(ti: RuntimeTaskInstance, log: Logger):
             next_method=next_method,
             trigger_timeout=timeout,
         )
-        SUPERVISOR_COMMS.send_request(msg=msg, log=log)
     except AirflowSkipException:
-        ...
+        msg = TaskState(state=TerminalTIState.SKIPPED)
```

Comment:
This path is not exercised in tests

Reply:
:( -- you aren't going to let me sneak that one in!

Reply:
Soz not soz

Reply:
Skipped the skipped state change 😆